lib/helpers/model/castBulkWrite.js

```diff
@@ -66,7 +65,9 @@ module.exports = function castBulkWrite(originalModel, op, options) {
           applyTimestampsToUpdate(now, createdAt, updatedAt, op['updateOne']['update'], {});
         }
 
-        applyTimestampsToChildren(now, op['updateOne']['update'], model.schema);
+        if (op['updateOne'].timestamps || op['updateOne'].timestamps == null) {
```

Comment:
Nitpick:
```suggestion
        if (op['updateOne'].timestamps === true || op['updateOne'].timestamps == null) {
```

Reply:
Can we do `if (op['updateMany'].timestamps !== false)` to be consistent with the other `if` statements in this file?